src/styles/main.scss

```diff
@@ -1,3 +1,72 @@
-body {
+$cell_size : 100px;
+$start-day : ( 
+  "tue" : 1,
+  "wed" : 2,
+  "thu" : 3,
+  "fri" : 4,
+  "sat" : 5,
+  "sun" : 6
+);
+
+.body {
   margin: 0;
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+.calendar {
+  padding: 10px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  align-items: center;
+  width: $cell_size * 7 + 6;
+  text-align: center;
+
+  @each $day, $n in $start-day {
+    &--start-day-#{$day} &_cell:first-child {
+      margin-left: $n * ($cell-size + 1px);
+    }
+  }
+
+  @for $day from 1 through 31 {
+    &--#{$day} &_cell:nth-child(-n + #{$day}) {
+      display: flex;
+    }
+  }
+
+  @for $index from 1 through 31 {
+    &_cell:nth-child(#{$index})::before {
+      content: "#{$index}";
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: 30px;
+    }
+  }
+
+  &_cell {
+    width: $cell_size;
+    height: $cell_size;
+    background-color: #eee;
+    border: 1px solid black;
+    box-sizing: border-box;
```

Comment:
remove it to `*` selector

Reply:
I dont understand what exactly i need to remove from there, because if i will remove selected lines, it will break all.

Reply:
box-sizing